src/azure-cli/azure/cli/command_modules/appservice/custom.py

```diff
@@ -3405,12 +3405,13 @@ def add_hc(cmd, name, resource_group_name, namespace, hybrid_connection, slot=No
                           send_key_name="defaultSender",
                           send_key_value=hy_co_keys.primary_key,
                           service_bus_suffix=".servicebus.windows.net")
+
     if slot is None:
         return_hc = web_client.web_apps.create_or_update_hybrid_connection(resource_group_name, name, namespace,
                                                                            hybrid_connection, hc)
     else:
         return_hc = web_client.web_apps.create_or_update_hybrid_connection_slot(resource_group_name, name, namespace,
-                                                                                hybrid_connection, hc, slot)
+                                                                                hybrid_connection, slot, hc)
```

Comment:
can we use _generic_site_operation here as well?

Reply:
I couldn't since this uses RG, name & and 2 additional properties which doesn't match the signature of the generic_site_operation.

Reply:
avoiding positional arguments could prevent such issue